<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrandTabs</class>
 <widget class="QMainWindow" name="BrandTabs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabBrands">
       <attribute name="title">
        <string>Rūšys</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="brandListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCountries">
       <attribute name="title">
        <string>Šalys</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="countryListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTags">
       <attribute name="title">
        <string>Tipai</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListView" name="tagListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
